packages/integrations/src/extraerrordata.ts

```diff
@@ -117,6 +125,12 @@ function _extractErrorData(error: ExtendedError): Record<string, unknown> | null
       extraErrorInfo[key] = isError(value) ? value.toString() : value;
     }
 
+    // Error.cause is a standard property that is non enumerable, we therefore need to access it separately.
+    // https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Error/cause
+    if (captureErrorCause && error['cause']) {
```

Comment:
```suggestion
    if (captureErrorCause && error.hasOwnProperty('cause')) {
```

Reply:
We cannot do this because `cause` is a prototype (? or class field, or getter idk) and hasOwnProperty will return false even though it is there.